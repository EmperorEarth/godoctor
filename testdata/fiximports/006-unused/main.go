package main // <<<<< fiximports,1,1,1,1,fail

// bytes and reflect are unused

import "bytes"
import "fmt"
import "math/rand"
import "reflect"

func main() {
  r := rand.New(rand.NewSource(99))
  fmt.Println(r.Float32())
}
