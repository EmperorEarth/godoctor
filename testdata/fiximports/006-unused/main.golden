package main // <<<<< fiximports,1,1,1,1,fail

// bytes and reflect are unused

import (
import "fmt"
import "math/rand"
)

func main() {
  r := rand.New(rand.NewSource(99))
  fmt.Println(r.Float32())
}
